syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.test.testbed";
option java_outer_classname = "TestDedProto";
option objc_class_prefix = "HLW";

package testbed;

service Testbed {
  // Sends a image flash request
  rpc FlashImage (ImageFlashRequest) returns (ImageFlashReply) {}
}

// The request message containing flashing image
message ImageFlashRequest {
  string file_path = 1;
  repeated uint32 serial_number = 2;
}

// The response message containing the flashing image
message ImageFlashReply {
  string message = 1;
}
